day_two_guide/topics/proc_restoring-etcd.adoc

```diff
@@ -252,3 +259,48 @@ ifeval::["{context}" == "downgrade"]
 # journalctl -fu etcd.service
 endif::[]
 ----
+
+== Restoring etcd on a static pod
+
+. Stop the etcd pod by moving the pod manifest YAML file to another directory:
++
+----
+$ mv /etc/origin/node/pods/etcd.yaml .
+----
+
+. Clear all old data:
++
+----
+$ rm -rF /var/lib/etcd
+----
++
+You use the etcdctl to recreate the data in the node where you restore the pod.
+
+. Restore the etcd snapshot to the mount path for the etcd pod:
++
+----
+$ export ETCDCTL_API=3
+$ etcdctl snapshot restore /backup/etcd/snapshot.db 
```

Comment:
This is called out in other documents (https://github.com/openshift/openshift-docs/pull/12322/files#diff-822c7f3ac6e45a71e6267af9592b9b5cR59) but you might want to link or re-call this out.